src/js/utils/UserSettings.js

```diff
@@ -39,7 +40,8 @@
         maintainRatio : true,
         resizeContent : false,
         origin : 'TOPLEFT'
-      }
+      },
+      COLOR_FORMAT: 0,
```

Comment:
I'm being super nitpicky, but I would prefer if the preference was a readable key 'hex', 'rgb', and  if the constant was a map:

```javascript
COLOR_FORMATS: {
  'hex' : 'hex',
  'rgb' : 'rgb'
}
```